### DISCUSSION TITLE
4.16.2: voting has stopped working for me on iOS 17.7

### DISCUSSION THREAD

**Discussion by e40** [dangwu/Octal#321] [Q&A]
comments and stories.  All give me: Something went wrong, Please try again later (OK).

Is this a known issue?  Been going on for more than a week for me.
In case it matters, I have this apps installed also:
* 1Blocker
* AdBuard
* BlockBear

**Comment by e40** [comment]
Just saw the post about the future of Octal.

**Reply by dangwu** [reply]
It should be fixed now with the 4.16.3 update today. Let me know if it persists
